packages/block-library/src/latest-posts/edit.js

```diff
@@ -72,47 +70,97 @@ class LatestPostsEdit extends Component {
 		this.isStillMounted = false;
 	}
 
-	toggleDisplayPostDate() {
-		const { displayPostDate } = this.props.attributes;
+	updateAttribute( name, value ) {
```

Comment:
@AmartyaU do we need this generic setter? It feels much better to have either the attribute set with an arrow functioni in the rendered component, or a function named specifically for what it does. I understand the need for DRY but sometimes this creates new API by mistake (e.g. we have setAttributes in the block API and it does not need an updateAttribute)

e.g. instead of 
`( value ) => this.updateAttribute( 'displayPostDate', value )` just do like you already do in other places `onChange={ ( value ) => setAttributes( { displayPostDate: value } ) }` and simply get rid of `updateAttribute` as it is useless.

Reply:
I was using updateAttribute because it was left over in the code when I saw it, so I thought it might be a coding standard and was thus using it. I also prefer doing it in place so will make the changes.

Reply:
Updated!